utils/pub/pubDates.ts

```diff
@@ -85,3 +85,13 @@ export const getPubCopyrightYear = (
 	const pubPublishedDate = getPubPublishedDate(pub);
 	return pubPublishedDate ? dateFormat(pubPublishedDate, 'yyyy') : dateFormat('yyyy');
 };
+
+export const getPubSubmissionDate = (pub: Pub) => {
+	if (pub.submission?.submittedAt) {
+		// This is a date string representing a time at midnight UTC for a given date.
```

Comment:
This is a good comment but I actually think (a variation of) it should live in `getLocalDateMatchingUtcCalendarDate`.